notebooks/02_model/lightfm_deep_dive.ipynb

```diff
@@ -0,0 +1,1505 @@
+{
```

Comment:
<p>just a reminder to remove this content when the notebook is finished</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1096/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>